src/device/device-management.ts

```diff
@@ -119,7 +210,7 @@ export async function getDevices(deviceKind?: media.DeviceKind): Promise<MediaDe
  *
  * @returns List of microphone devices in an array of MediaDeviceInfo objects.
  */
-export async function getAudioInputDevices(): Promise<MediaDeviceInfo[]> {
+export async function getMicrophones(): Promise<MediaDeviceInfo[]> {
```

Comment:
I kinda feel like webrtc-core should use the vocabulary of the underlying API (devices).  (Here and other naming above)

Reply:
we could but again the SDK have to add a wrapper to simply map the function name , simply more work , 

as the name is more internal can we re consider it